var/spack/repos/builtin/packages/comd/package.py

```diff
@@ -56,7 +59,7 @@ def build_targets(self):
                 comd_variant += '-mpi'
                 targets.append('CC = {0}'.format(self.spec['mpi'].mpicc))
             else:
-                targets.append('CC = {0}'.format('spack_cc'))
+                targets.append(f'CC = {spack_cc}')
```

Comment:
Please use the older `str.format` or `%s` format for strings. Python `f-strings` support began only with python3.6, and Spack supports python2.6, python2.7, and python3.5.

Reply:
Ping @jfalkner